tests/test_shell.py

```diff
@@ -269,8 +270,17 @@ def test_find(self):
         assert '{"a": 0}' in out
         assert '{"a": 2}' in out
 
+        job = project.open_job({"a": [0, 1, 0]})
+        msg = "python -m signac find a".split()
+        msg.append('[0, 1, 0]')
+        msg.append('--sp')
```

Comment:
Adding list after calling `split()` because this method splits up list element into strings, even when the whole list is in quotations. (Is there more elegant solution?)

Reply:
No, there's not really a more elegant solution without doing more work than you want to. The solutions that I can think of would involve using `re.split` or `csv.reader`, both of which seem pretty heavyweight for this purpose.